e2e/kind.sh

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env bash
 # shellcheck disable=SC2034
-KUBECONFIG="kind.yaml"
+export KUBECONFIG="kind.yaml"
```

Comment:
Without exporting this, the variable was not available in subshells, which mean that the different commands were actually using `$HOME/kube/config.yaml`

Reply:
We didn't invoke any subshells, right?

Reply:
hm not sure... but I checked and `kind` and `kubectl` were not using this file at all. There was no file called kind.yaml during the tests. Only now, after exporting this, does `kind` create the file.

Reply:
weird it worked and still works without the export on my laptop. The kind.yaml file is created and located at e2e/kind.yaml

Reply:
For me it always "worked" but just didn't use that file

Reply:
Hmmmmmm I agree that needing to export is weird. I'll post a test PR tomorrow that removes the `export` so that we can test that everything works and get down to the bottom of this. Sounds good?